src/getNativeComponent.tsx

```diff
@@ -0,0 +1,86 @@
+import {NativeModule} from 'react-native';
+import {
+  find,
+  NATIVE_COMPONENTS,
+  UNSUPPORTED_PAYMENT_METHODS,
+} from './ComponentMap';
+import {
+  UNKNOWN_PAYMENT_METHOD_ERROR,
+  UNSUPPORTED_PAYMENT_METHOD_ERROR,
+} from './Core/constants';
+import {PaymentMethod, PaymentMethodsResponse} from './Core/types';
+import {AdyenNativeComponentWrapper} from './AdyenNativeComponentWrapper';
+import {
+  AdyenActionComponent,
+  AdyenDropIn,
+  AdyenApplePay,
+  AdyenGooglePay,
+  AdyenInstant,
+} from './AdyenNativeModules';
+
+/**
+ * Get native component capable of handling provided payment method type.
+ */
+export function getNativeComponent(
+  typeName: string,
+  paymentMethods: PaymentMethodsResponse,
+): {
+  nativeComponent: AdyenActionComponent & NativeModule;
+  paymentMethod: PaymentMethod | undefined;
```

Comment:
I think we can define `paymentMethod: PaymentMethod | undefined;` as optional? `paymentMethod?: PaymentMethod;`

Reply:
Fixed [0cb5a11](https://github.com/Adyen/adyen-react-native/pull/341/commits/0cb5a115d0c3a9f599933c4ab96045713f5ceb49)